### ISSUE TITLE
Image upload fails on apply and reload or save

### ISSUE THREAD

**Issue by iamrobert** [FLEXIcontent/flexicontent-cck#982]
Hi,

When I got to apply and save images it fails.

Issue:
https://www.youtube.com/watch?v=Zx9PjcC06kE

In this file I noticed:
https://github.com/FLEXIcontent/flexicontent-cck/blob/master/plugins/flexicontent_fields/image/image.php

```
$img_link = htmlspecialchars(phpThumbURL(
						'src=' . $img_link . '&w=' . $preview_thumb_w . '&h=' . $preview_thumb_h . '&zc=1&q=95&ar=x' . $f,
						'/j3/components/com_flexicontent/librairies/phpthumb/phpThumb.php'
					));
```

contains: /j3/components - and I changed it to universal but no difference.

I'm getting the following error:

/administrator/index.php?option=com_flexicontent&task=filemanager.uploads&2092cfafb3b9f60b048e60e05a135311=1&fieldid=32&u_item_id=1%20500%20(Internal%20Server%20Error)

WIth the following messages:
https://i.imgur.com/DRPBohV.png

Any Ideas?

**Comment by iamrobert** [comment]
Also this file gives an error:

/administrator/index.php?option=com_flexicontent&task=filemanager.uploads&history=1&2092cfafb3b9f60b048e60e05a135311=1&fieldid=36&u_item_id=12

Error Message:
{"jsonrpc" : "2.0", "error" : {"code": status, "message": "Filename has invalid characters (or other error occured)"}, "data" : null}

**Comment by ggppdk** [comment]
version you are using , or which github branch ?

**Comment by ggppdk** [comment]
Yes this was bug due to a recent commit b5f925e
fixed with 5952d71
Thanks !

**Comment by iamrobert** [comment]
I am getting the following issues now - since trying:

https://github.com/FLEXIcontent/flexicontent-cck/releases/tag/v3.3.5

Here's a video of the issue:

https://www.youtube.com/watch?v=eg23v9WyL78


**500 ERROR:**
POST http://localhost/iam/administrator/index.php?option=com_flexicontent&task=filemanager.uploads&f75a43c044e0d45b7719ecf138a6b294=1&fieldid=32&u_item_id=2 500 (Internal Server Error)

![Image](https://i.imgur.com/XjROybt.png)

JS Issue:
![Image](https://i.imgur.com/pG6P97p.png)

Could you let me know why?
